### PR TITLE
Add Windows Installer support

### PR DESCRIPTION
**Pull request by cprecioso**
You'll need to `brew install mono wine` in your computer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "author": {
     "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
-    "url": "sindresorhus.com"
+    "url": "http://sindresorhus.com"
```
Comment (cprecioso) [inline package.json]: This is actually needed or `electron-installer-windows` will complain about a malformed URL

**Comment by cprecioso** [comment]
It seems this should be built on Windows, sorry. Are you able to build this on Windows yourself, or willing to set up AppVeyor for building?

**Comment by cprecioso** [comment]
BTW closes #63

**Comment by sindresorhus** [comment]
Thanks, but I plan on replacing the current system with `electron-builder` very soon.

**Pull request closed without merging**